otherlibs/unix/wait.c

```diff
@@ -77,9 +67,7 @@ CAMLprim value unix_wait(value unit)
   return alloc_process_status(pid, status);
 }
 
-#if defined(HAS_WAITPID) || defined(HAS_WAIT4)
-
-#ifndef HAS_WAITPID
+#ifdef HAS_WAIT4
 #define waitpid(pid,status,opts) wait4(pid,status,opts,NULL)
```

Comment:
When `waitpid` and `wait4` are both available, the old code used `waitpid` while the new uses `wait4`. Why not always use the standard `waitpid` and drop the detection logic for `wait4`?

Reply:
Oops, that’s a mistake - as you say, `wait4` is no longer required